OmniCore.Radio/OmniCore.Radio.RileyLink/RileyLink.cs

```diff
@@ -54,7 +54,7 @@ private RileyLinkStatistics GetStatistics(IMessageExchangeProgress messageProgre
 
         public RileyLink(ErosRadioPreferences erosRadioPreferences)
         {
-            TxAmplification = TxPower.A4_Normal;
+            TxAmplification = TxPower.A5_High; // On initialisation, start with some extra TxPower (it will auto adjust)
```

Comment:
I don't see how this is justified, high tx is one of the primary reasons for pod failures as it leads to broken comms

Reply:
Considered this as safe:
1. Autoadjust will take it back to to normal or lower after initial comms?
2. Effectively it is identical to what I do when receiving errors on Pod comminucations: bringen RL (much) closer to the Pod?

Also: Autoadjust starts lowering transmission level on Pod communication errors - which I find contra-intuitive. Maybe start at a bit higher level to make sure range problems have less effect on the  initialization fase?